src/redirects.ts

```diff
@@ -131,7 +131,15 @@ export async function redirects(request: Request) {
     // See https://github.com/serlo/serlo.org-cloudflare-worker/issues/184
     // Can be deleted after a while after the /entity/view/<id>/toc route
     // got deleted
-    const match = /^\/entity\/view\/(\d+)\/toc$/.exec(url.pathname)
+    let match = /^\/entity\/view\/(\d+)\/toc$/.exec(url.pathname)
+
+    if (match) {
+      url.pathname = `/${match[1]}`
+
+      return url.toRedirect(308)
```

Comment:
308 vs 301 is quite the debate.
and just by habit and also after reading some of the debate I'd use 301 for this route since we expect GET requests.